docs/mechanisms/build.md

```diff
@@ -33,6 +33,18 @@ Moonshot.config do |c|
 ...
 ```
 
+**skip_ci_status** is an optional flag. It would allow us to skip checks 
+on the commit's CI job status. If its not finished yet, no need for waiting till timeout.
```

Comment:
How about instead of "If...", we say "Without this option, the GithubRelease mechanism will wait until the build is finished."